src/components/Visualizations/inflictorWithValue.jsx

```diff
@@ -39,7 +40,9 @@ const tooltipContainer = thing => (
       <div
         dangerouslySetInnerHTML={{
           __html: thing.cmb
-            .replace('/apps/dota2/images/tooltips/mana.png', `${API_HOST}/apps/dota2/images/tooltips/mana.png`)
+            .replace('src="/apps/dota2/images/tooltips/mana.png"', `src="${API_HOST}/apps/dota2/images/tooltips/mana.png"`)
```

Comment:
This should now be a single line that does:
`.replace(/http:\/\/cdn\.dota2\.com/g, '');`